utils/cluster_manager.py

```diff
@@ -283,26 +283,41 @@ def print_servers_json(servers: List[Server]):
     print("SERVERS_JSON={}".format(json.dumps(arr)))
 
 
-def next_free_port(
-    min_port: int = 6379, max_port: int = 55535, timeout: int = 60
-) -> int:
+def approximate_ports_in_use() -> int:
+    """
+    Returns a rough count of lines from `netstat -an`.
+    Enough to see if we're in the hundreds or thousands.
+    """
+    output = subprocess.check_output(["netstat", "-an"])
+    # or ["ss", "-an"] if `netstat` isn't installed
+    lines = output.decode().splitlines()
+    return len(lines)
+
+def next_free_port(min_port: int = 6379, max_port: int = 55535, timeout: int = 60) -> int:
+    # Log how many sockets show up right now
+    num_sockets = approximate_ports_in_use()
```

Comment:
It is not in use anywhere. Is it intended? Please check that you're not missing commited changes.